service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GpcWebClientFilter.java

```diff
@@ -14,7 +14,9 @@ public ExchangeFilterFunction errorHandlingFilter() {
         return ExchangeFilterFunction.ofResponseProcessor(clientResponse -> {
             clientResponse.statusCode();
             if (clientResponse.statusCode().is5xxServerError()
-                || clientResponse.statusCode().is4xxClientError()) {
+                || clientResponse.statusCode().is4xxClientError()
+                || clientResponse.statusCode().is3xxRedirection()
+                || clientResponse.statusCode().is1xxInformational()) {
```

Comment:
@mattd-kainos, do we need to also return errors if we encounter "1xx" response codes? Seems pretty unlikely for them to appear.